CONTRIBUTING.md

```diff
@@ -304,9 +304,54 @@ verified to be green within CI before merging to main. Merging to main
 automatically triggers publishing of artifacts and deployment of the software to
 production environments such as the Pathling website and sandbox instance.
 
-### Coding conventions
 
-#### Comments
+## Commit Message Format
+
+Write commit messages that capture the **objective** of the change, not the specific implementation details that can be obtained from the diff.
+
+**Structure**:
+```
+<type>: <succinct description of the objective>
+
+<optional body explaining the why and context>
+```
+
+**Types**:
+- `fix:` - Bug fixes or resolving warnings/errors
+- `feat:` - New features or enhancements
+- `refactor:` - Code restructuring without changing behavior
+- `docs:` - Documentation updates
+- `test:` - Test-related changes
+- `chore:` - Build, tooling, or dependency updates
```

Comment:
I must confess to not immediately loving this convention, but seeing as you are the first to propose one I will go with it 🙂